app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -186,30 +186,26 @@ class RecentlyPlayedFragmentTest {
   fun testRecentlyPlayedTestActivity_sectionDividerIsNotDisplayed() {
     fakeOppiaClock.setFakeTimeMode(FakeOppiaClock.FakeTimeMode.MODE_UPTIME_MILLIS)
     storyProgressTestHelper.markRecentlyPlayedFractionsStory0Exp0(
-      profileId = profileId,
+      profileId,
       timestampOlderThanOneWeek = false
     )
     storyProgressTestHelper.markRecentlyPlayedRatiosStory0Exp0(
-      profileId = profileId,
+      profileId,
       timestampOlderThanOneWeek = true
     )
     ActivityScenario.launch<RecentlyPlayedActivity>(
       createRecentlyPlayedActivityIntent(
-        internalProfileId = internalProfileId
+        internalProfileId
       )
     ).use {
       testCoroutineDispatchers.runCurrent()
-      onView(withId(R.id.ongoing_story_recycler_view)).perform(
+      onView(withId(R.id.promoted_story_recycler_view)).perform(
         scrollToPosition<RecyclerView.ViewHolder>(
           0
         )
       )
       onView(
-        atPositionOnView(
-          recyclerViewId = R.id.ongoing_story_recycler_view,
-          position = 0,
-          targetViewId = R.id.divider_view
-        )
+        atPositionOnView(R.id.promoted_story_recycler_view, 0, R.id.divider_view)
```

Comment:
```suggestion
        atPositionOnView(
          recyclerViewId = R.id. promoted_story_recycler_view,
          position = 0,
          targetViewId = R.id.divider_view
        )
```

I think we can keep this unchanged.

Reply:
done